packages/desktop/views/update-stronghold/views/UpdateStrongholdView.svelte

```diff
@@ -33,8 +35,17 @@
                 await migrateStrongholdFromOnboardingProfile(password)
                 emitStrongholdMigrationEvent({ success: true, onboardingType })
             } else {
-                await migrateStrongholdFromActiveProfile(password)
-                emitStrongholdMigrationEvent({ success: true })
+                try {
+                    await migrateStrongholdFromActiveProfile(password)
+                    emitStrongholdMigrationEvent({ success: true })
+                } catch (err) {
```

Comment:
This will eat the error, which we still need for failed migrations that indeed failed for another reason, we need to throw an excepcion for non v3 profiles that still failed 🙏🏼